### ISSUE TITLE
not working properly 

### ISSUE THREAD

**Issue by suriyaJaay** [Idered/Editr.js#34]
not working properly even though i followed the document.kindly see the bellow codes
`<!DOCTYPE html>
<html>
<head>
<link rel="stylesheet" href="/editr/editr.css">
	<title></title>
</head>
<body>

<script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
<script src="//cdn.jsdelivr.net/ace/1.1.01/min/ace.js"></script>
<script src="//cdn.jsdelivr.net/ace/1.1.01/min/ext-emmet.js"></script>
<script src="/editr/libs/ext.emmet.js"></script>
<script src="/editr/editr.js"></script>

<div class="editr" data-item="PROJECT-NAME" data-files-html="index-1.html;index-2.html" data-files-css="!normalize.css;style.css" data-files-js="!jquery.js;script.js"></div>
</body>

<script type="text/javascript">
	$('.editr').each(function() {
    new Editr({ el: this });
});
</script>
</html>

`
